CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 1.0.4
+
+- Fix validation failing when subdomain entered as fully-qualified domain name.
+- Increase maximum supported WordPress version.
```

Comment:
Please include reference to issues (or pull requests) here.

Reply:
Added eab7042